styles.css

```
@@ -9936,3 +10036,514 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
+    flex-direction: column;
+    gap: 0.95rem;
+    justify-self: center;
+    transform: translateX(clamp(0.75rem, 1.2vw, 1.4rem));
+}
+
+#fullscreen-cover-overlay .fullscreen-artwork-card {
+    width: min(420px, 100%);
+    aspect-ratio: 1 / 1;
+    border-radius: 18px;
+    overflow: hidden;
+    box-shadow: 0 28px 80px rgba(0, 0, 0, 0.26);
```

Stylelint (17.6.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)